pyne/fortranformat/_parser.py

```diff
@@ -79,7 +79,7 @@ def _parse_tokens(tokens, reversion=False, version=None):
         elif ed_type == 'ED10':
             ed = _read_ed10(token_set)
         else:
-            raise InvalidFormat('Could not identify edit descriptor in sequence $s' % str(token_set))
+            raise InvalidFormat('Could not identify edit descriptor in sequence %s' % str(token_set))

```

Comment:
I think the OP's suggestion of switching from the `%`-formatting to `string.format()` is probably a good idea here.

Reply:
I can do that throughout the file. Should I scrape other files for that or will that take me weeks?

Reply:
> Thanks for picking up this pinoli @opotowsky

I drink piñon coffee every day, so I'm devoted to all low-hanging pinoli!

Reply:
(No idea how I responded to that here)